openshift/ci-operator/build-image/Dockerfile

```diff
@@ -1,17 +1,23 @@
-# Dockerfile to bootstrap build and test in openshift-ci
+# DO NOT EDIT! Generated Dockerfile.
 
+# Dockerfile to bootstrap build and test in openshift-ci
 FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17 as builder
 
-# Add kubernetes repository
-ADD openshift/ci-operator/build-image/kubernetes.repo /etc/yum.repos.d/
+RUN echo "[kubernetes]" >> /etc/yum.repos.d/kubernetes.repo && \
+    echo "name=Kubernetes" >> /etc/yum.repos.d/kubernetes.repo && \
+    echo "baseurl=https://pkgs.k8s.io/core:/stable:/v1.29/rpm/" >> /etc/yum.repos.d/kubernetes.repo && \
+    echo "enabled=1" >> /etc/yum.repos.d/kubernetes.repo && \
+    echo "gpgcheck=1" >> /etc/yum.repos.d/kubernetes.repo && \
+    echo "gpgkey=https://pkgs.k8s.io/core:/stable:/v1.29/rpm/repodata/repomd.xml.key" >> /etc/yum.repos.d/kubernetes.repo
 
 RUN yum install -y kubectl httpd-tools
 
-# Serverless-Operator `make generated-files` needs helm
-RUN curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+RUN wget https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 && \
+    chmod 700 ./get-helm-3
+
+RUN ./get-helm-3 --version v3.11.3 --no-sudo && helm version
 
 RUN GOFLAGS='' go install github.com/mikefarah/yq/v3@latest
-RUN GOFLAGS='' go install knative.dev/test-infra/tools/kntest/cmd/kntest@latest
```

Comment:
/hold

Will test this in a separate CI because this change will be only applied after merging the current PR, and then break future PRs.

Reply:
Testing in https://github.com/openshift/release/pull/56274